content/webapp/pages/exhibition-guide.tsx

```diff
@@ -309,27 +317,26 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
                 {type === 'audio-with-descriptions' &&
                   audioWithDescription?.url && (
                     <AudioPlayer
-                      title={
-                        stop.title
-                          ? `${number}. ${stop.title}`
-                          : `${number}. ${standaloneTitle}`
-                      }
+                      title={getTitle(stop)}
                       audioFile={audioWithDescription.url}
                     />
                   )}
                 {type === 'audio-without-descriptions' &&
                   audioWithoutDescription?.url && (
                     <AudioPlayer
-                      title={
-                        stop.title
-                          ? `${number}. ${stop.title}`
-                          : `${number}. ${standaloneTitle}`
-                      }
+                      title={getTitle(stop)}
                       audioFile={audioWithoutDescription.url}
                     />
                   )}
                 {type === 'bsl' && bsl.embedUrl && (
-                  <VideoEmbed embedUrl={bsl.embedUrl} />
+                  <figure className="no-margin">
+                    <Space v={{ size: 'm', properties: ['margin-bottom'] }}>
+                      <figcaption className={font('intb', 5)}>
+                        {getTitle(stop)}
+                      </figcaption>
+                    </Space>
+                    <VideoEmbed embedUrl={bsl.embedUrl} />
+                  </figure>
```

Comment:
minor: I wonder if it would be worth putting this in a `<VideoPlayer>` component to match AudioPlayer?